README.md

```diff
@@ -11,65 +11,32 @@ Supported features:
 - [ ] Data Availability Sampling
 - [ ] Creating, distributing, and listening for Fraud proofs
 
-## Building and running
+## Running the node
 
-Install common dependencies
+For detailed information about setup, building, and running Lumina node locally or in a browser see [lumina-cli](cli/README.md).
 
+### Run the node locally
 ```bash
-# install dependencies
-sudo apt-get install -y build-essential curl git protobuf-compiler
-
-# install rust
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
-
-# open a new terminal or run
-source "$HOME/.cargo/env"
-
-# clone the repository
-git clone https://github.com/eigerco/lumina
-cd lumina
-```
```

Comment:
I would leave the build from sources details in the main repo's readme but moved to a new section below, like build from sources. Here and in `cli` I would just keep the `cargo install lumina-cli --locked` step.

Reply:
I see, the distinction makes sense, moved all the development/source related notes to the main readme, leaving small running notes in `lumina-cli` 👍